vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -184,11 +184,10 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 	// DEPRECATED: remove and replace with something more accurate
 	loadingRules.DefaultClientConfig = &clientcmd.DefaultClientConfig
 
-	if clientcmd.UseOpenShiftKubeConfigValues {
-		flags.StringVar(&loadingRules.ExplicitPath, clientcmd.OpenShiftKubeConfigFlagName, "", "Path to the kubeconfig file to use for CLI requests.")
-	} else {
-		flags.StringVar(&loadingRules.ExplicitPath, "kubeconfig", "", "Path to the kubeconfig file to use for CLI requests.")
-	}
+	flags.StringVar(&loadingRules.ExplicitPath, clientcmd.OpenShiftKubeConfigFlagName, "", "Path to the kubeconfig file to use for CLI requests.")
```

Comment:
does binding the same flag twice actually work?  Which one wins?

Reply:
yes, multiple flags can bind to the same underlying value. last one parsed wins, just like if you specify the same flag multiple times:

```sh
oc --config foo.kubeconfig --config bar.kubeconfig get pods
error: stat bar.kubeconfig: no such file or directory
```